CHANGELOG.md

```diff
@@ -60,6 +60,7 @@ To learn more about active deprecations, we recommend checking [GitHub Discussio
 
 - TODO ([#XXX](https://github.com/kedacore/keda/issues/XXX))
 - **General**: Declarative parsing of scaler config ([#5037](https://github.com/kedacore/keda/issues/5037))
+- *ActiveMQ Scaler*: Refactor ActiveMQ config parser ([#5797](https://github.com/kedacore/keda/issues/5797))
```

Comment:
Please move this to `Other` section, as it is not user facing change. Thanks!

Reply:
Or ideally create one entry in the changelog for all scalers.

Reply:
> Or ideally create one entry in the changelog for all scalers.

I like that option best, probably no need to pollute changelog with bunch of similar refactors. Given https://github.com/kedacore/keda/pull/5676 already added an entry to changelog, I guess that should be sufficient?

Reply:
Yeah, the only thing we can do is to also add this https://github.com/kedacore/keda/issues/5797 to that changelog entry, something like this `(#5037] | #5797 )`